posts/2021-12-13-log4j2-response.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Vendia's Response to CVE-2021-44228"
+description: "A note on the impact to Vendia and our customers"
+date: '2021-12-13'
+authors:
+- James Gimourginas
+categories:
+- announcements
+---
+
+Vendia is aware of the recently disclosed [security issue](CVE-2021-44228) relating to the open-source Apache “Log4j2" library.
```

Comment:
```suggestion
Vendia is aware of the recently disclosed [security issue](https://nvd.nist.gov/vuln/detail/CVE-2021-44228) relating to the open-source Apache “Log4j2" library.
```

Reply:
Updated.  Not sure where that link went :).